Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/RelationshipsStepDefinitions.cs

```diff
@@ -187,6 +219,23 @@ public async Task ThenTheResponseContainsARelationship()
         }
     }
 
+    [Then("a relationship can be established")]
+    public void ThenARelationshipCanBeEstablished() => PerformCanEstablishRelationshipCheck(true);
+
+    [Then("a relationship can not be established")]
+    public void ThenARelationshipCanNotBeEstablished() => PerformCanEstablishRelationshipCheck(false);
+
+    private void PerformCanEstablishRelationshipCheck(bool value)
+    {
+        if (_canEstablishResponse != null)
+        {
+            _canEstablishResponse!.Should().BeASuccess();
```

Comment:
This assertion is redundant, right? Because in another `Then` step we check for the status code.